io.openems.edge.bridge.http/src/io/openems/edge/bridge/http/api/HttpMethod.java

```diff
@@ -0,0 +1,24 @@
+package io.openems.edge.bridge.http.api;
+
+public enum HttpMethod {
+	GET(false), //
+	POST(true), //
+	PUT(true), //
+	DELETE(false), //
+	PATCH(true), //
+	HEAD(false), //
+	OPTIONS(false), //
+	TRACE(false) //
+	;
+
+	private final boolean bodyAllowed;
```

Comment:
@michaelgrill I would just make this `public final boolean isBodyAllowed;` - and avoid the getter.